tests/testthat/test-with.R

```diff
@@ -44,7 +44,7 @@ test_that("all default linters are tagged default", {
   skip_on_covr()
 
   expect_identical(linters_with_tags("default"), linters_with_defaults())
-  expect_length(linters_with_tags("default", exclude_tags = "default"), 0L)
+  expect_length(linters_with_tags(NULL, exclude_tags = available_tags()), 0L)
```

Comment:
It might be nice to support `linters_with_tags(exclude_tags = ...)` without specifying `tags = NULL`.

That would require setting a default of `tags = NULL` (or hacking around with `missing()`), but I hesitated since this would allow `linters_with_tags()` (i.e., no arguments) to work, which doesn't quite read as well as `linters_with_tags(NULL)`.

We could allow both behaviors ([1] require at least one argument specified and [2] allow only `exclude_tags` to be specified), again if we hack around with `missing()`. I held off on that for now since I'm recently more loath to fidget with `missing()`.